src/meta/server_state.cpp

```diff
@@ -2864,5 +2883,79 @@ void server_state::clear_app_envs(const app_env_rpc &env_rpc)
                    new_envs.c_str());
         });
 }
+
+namespace {
+
+bool validate_target_max_replica_count_internal(int32_t max_replica_count,
+                                                int32_t alive_node_count,
+                                                std::string *hint_message)
+{
+    dassert_f(FLAGS_min_allowed_replica_count <= FLAGS_max_allowed_replica_count,
+              "min allowed replica count({}) should be <= max allowed replica count({})",
+              FLAGS_min_allowed_replica_count,
+              FLAGS_max_allowed_replica_count);
+
+    if (alive_node_count < 1) {
```

Comment:
Meta server has a state called `freeze`, if alive node is not enough, meta server will freeze itself, in this situation, `create_app` will be rejected. 
https://github.com/XiaoMi/rdsn/blob/289eb4609ae781e3bcc6bbc6bfeae64dfd8fa785/src/meta/meta_service.cpp#L97
You can add meta_function_level on the top of `create_app`, like:
https://github.com/XiaoMi/rdsn/blob/289eb4609ae781e3bcc6bbc6bfeae64dfd8fa785/src/meta/meta_service.cpp#L724-L725